README.md

```diff
@@ -7,6 +7,14 @@ Using this API client requires an active account. [Sign up for a Bitmovin API ke
 
 The full [Bitmovin API reference](https://bitmovin.com/encoding-documentation/bitmovin-api/) can be found on our website.
 
+## Deprecation Notice
+
+`bitmovin-ruby` is the legacy Bitmovin API client for Ruby and is not maintained anymore.
+
+We recommend using one of the newer clients, you can find a list of available clients in our [Bitmovin Docs](https://developer.bitmovin.com/encoding/docs/sdks). 
```

Comment:
funny, there is no ruby client, do you recommend to change the language?
think it will be easier to say good bye to bitmovin